### ISSUE TITLE
Allow multithreaded SVG-Generation

### ISSUE THREAD

**Issue by NiklasRentzCAU** [kieler/KLighD#15]
The offscreen rendering of lots of graphs takes a considerable amount of time. It would be great to have the option to do a true parallel generation of SVG images.

This requires changes to KlighD, especially in the area of size estimation, as well as ELK, at least at the progress monitor handling.

**Comment by NiklasRentzCAU** [comment]
This issue is migrated from the old KIELER Pragmatics JIRA issue tracker. See the old discussion on this ticket in [KIPRA-1896](https://rtsys.informatik.uni-kiel.de/jira/projects/KIPRA/issues/KIPRA-1896).

**Comment by DerStoffel** [comment]
Hi everyone,

is there any update? I would have loved to read the history of this ticket, but can not access your Jira.
This is still an ongoing topic for us :)

Kind regards.

**Comment by NiklasRentzCAU** [comment]
Our old Jira has been migrated here and shut down now. The message here is just under every migrated ticket. I cannot  find out anymore either if there was much more discussion around this ticket, but I would guess there was not much discussion. This is one of the tickets in our backlog and more of a "won't do" ticket for myself. To generate multiple SVGs in parallel, just scripting to call the SVG generation multiple times in parallel would be a fine way to go as far as I see this issue with no pressing need to directly implement such functionality in KLighD itself right now. 
Do you have a different use case for this where a solution in KLighD would be sensible?
